src/html_support_files/odoc.css

```diff
@@ -782,20 +784,18 @@ td.def-doc *:first-child {
   line-height: 1.2;
 }
 
-:root {
-}
 
 .odoc-toc {
-  --toc-top: 20px;
-
+  /* TODO : make this smaller if there is no search bar */
+  --toc-top: calc(var(--search-bar-height) + var(--search-padding-top) + 20px);
```

Comment:
Is the search bar sticky as well ? If not, this is not needed at all, `--toc-top: 20px` is sufficient.

Reply:
>  is not properly spaced from the content is hovers
i do not understand this part
> When the search bar is not present, this calculation adds space above the TOC that doesn't look good either.
I can fix this.
> It's only surrounded by a white shadow that makes it hard to spot where the contents begins 
This is probably fixable, although I do not remember having this issue
>  doesn't fit with Odoc's theme.
Its true the input element is practically unstyled, but I plan to style it in my colors pr. As of now, it is dependent on browser defaults, which are very different and sometimes ugly.

I like the sticky sidebar, for the reason that I think that searching for something while looking at the doc is very useful. If it is not sticky, if you read the doc and the want to search for something, then you have to scroll to the top, and might forget what you wanted to do, and you also cannot type the query while looking at the doc (for instance copying a typename in a type-search query).
Also, the sidebar was not made sticky by this PR, and changing it is a little out of scope.

Reply:
> is not properly spaced from the content is hovers

I mean that it's hard to distinguish where the content starts below the search bar. 

> If it is not sticky, if you read the doc and the want to search for something, then you have to scroll to the top, and might forget what you wanted to do, and you also cannot type the query while looking at the doc

I don't think this workflow is well implemented currently. The dropdown showing the search results is above the page and might interfere. On mobile, almost none of the page is visible while typing the search query.
This waste vertical space at all time, not just when I intent to search.

>  Also, the sidebar was not made sticky by this PR, and changing it is a little out of scope.

Agree! I just noticed that the search bar was sticky while reviewing this calculation. Let's not do change the search bar in this PR.

Reply:
> On mobile, almost none of the page is visible while typing the search query.
True, the search bar should not be sticky on mobile
> This waste vertical space at all time, not just when I intent to search.
That is true, there is a tradeoff here. Maybe with some js magic we can have both, but I am not sure.

Reply:
I would trade having to scroll to the top for more vertical space.